src/Sentry.Unity.Editor/Android/ProguardSetup.cs

```diff
@@ -60,12 +60,19 @@ public void AddToGradleProject()
             _logger.LogDebug("Writing proguard rule file to {0}.", ruleFile);
             File.Copy(Path.GetFullPath($"Packages/{SentryPackageInfo.GetName()}/Plugins/Android/{RuleFileName}"), ruleFile, true);
 
-            var gradleNew = Regex.Replace(gradle, @"(\s+consumerProguardFiles .*)", "$1, '" + RuleFileName + "'");
-            if (gradle.Length == gradleNew.Length)
+            if (gradle.Contains(RuleFileName))
             {
-                throw new Exception($"Couldn't add Proguard rule {RuleFileName} to {_gradleScriptPath} - no `consumerProguardFiles` found.");
+                _logger.LogDebug($"Proguard rule {RuleFileName} has already been added to {_gradleScriptPath} `consumerProguardFiles` in a previous build.");
+            }
+            else
+            {
+                var gradleNew = Regex.Replace(gradle, @"(\s+consumerProguardFiles [^\r\n]*)", "$1, '" + RuleFileName + "'");
+                if (gradle.Length == gradleNew.Length)
+                {
+                    throw new Exception($"Couldn't add Proguard rule {RuleFileName} to {_gradleScriptPath} - no `consumerProguardFiles` found.");
```

Comment:
Fixup the file so that this passes. I know that's not perfect, but what's the alternative? Producing a build that breaks at runtime and won't capture errors? That's arguable worse than disabling Sentry native support for Android